Fabric.Authorization.Persistence.SqlServer/Stores/SqlServerRoleStore.cs

```diff
@@ -30,8 +34,18 @@ public SqlServerRoleStore(IAuthorizationDbContext authorizationDbContext)
         }
 
         public async Task<Role> Get(Guid id)
+        {
+            var role = await GetEntityModel(id);
+            return role.ToModel();
+        }
+
+        public async Task<EntityModels.Role> GetEntityModel(Guid id)
         {
             var role = await _authorizationDbContext.Roles
+                .Include(r => r.RolePermissions)
+                .ThenInclude(rp => rp.Permission)
```

Comment:
Do we need to filter to eliminate deleted permissions and permissions that are deleted from the role? Same comment with groups.

Reply:
We do need filter out deleted RolePermission entities now that we're doing a soft delete.

Reply:
If the parent is not included in the result then I dont think the related entities will get loaded

Reply:
Not sure what you mean?